ps_wirepayment.php

```diff
@@ -129,6 +129,8 @@ protected function _postValidation()
                 $this->_postErrors[] = $this->trans('Account details are required.', [], 'Modules.Wirepayment.Admin');
             } elseif (!Tools::getValue('BANK_WIRE_OWNER')) {
                 $this->_postErrors[] = $this->trans('Account owner is required.', [], 'Modules.Wirepayment.Admin');
+            } elseif (!Tools::getValue('BANK_WIRE_ADDRESS')) {
+                $this->_postErrors[] = $this->trans('Account address is required.', [], 'Modules.Wirepayment.Admin');
```

Comment:
Do we really need the `elseif`? I guess all of these errors could be shown at the same time, it would avoid filling one field at a time to see errors

Reply:
@NeOMakinG You're right. It's fixed.